packages/fxa-settings/src/lib/config.ts

```diff
@@ -76,6 +76,7 @@ export interface Config {
     logPings: boolean;
     debugViewTag: string;
   };
+  redirectAllowList: string[];
```

Comment:
Nit: To be consistent with other existing naming conventions this should be `redirectAllowlist`.